### PR TITLE
Republican 7th ideas change

### PR DESCRIPTION
**Pull request by ComradDoge**
Republics do not deserve absolutism. For the time I thought corruption reduction should suffice but preferably something else should be as a final for republican ideas unless yall think this is fine too.

Also this might be a small change but the change in the reduction of brain cells trying to understand github is much higher

### CODE REVIEW

**Comment by stackpoint** [comment]
I saw your post on discord but forgot to note it down and the nature of the chat format meant it scrolled away.

I agree with the sentiment that it should be replaced but I don't think yearly corruption is a sufficient replacement. Unless republics generate a significant amount of corruption that would require it?

**Review by stackpoint — CHANGES_REQUESTED** [review]
I saw your post on discord but forgot to note it down and the nature of the chat format meant it scrolled away. It's a lot easier to keep track of issues on GitHub. GitHub can be a bit confusing to get started with but the collaboration element is unrivaled.

I agree with the sentiment that it should be replaced but I don't think yearly corruption is a sufficient replacement. Unless republics generate a significant amount of corruption that would require it?

**Comment by ComradDoge** [comment]
No worries regarding discord if anything forced me to start learning github.

Regarding corruption I will agree that it should be something else but I am not sure what. I was thinking of giving republics a 10% discount of promoting mercantilism but I am not sure how to do that (if that even exists)

**Comment by stackpoint** [comment]
Here's a list of modifiers: https://eu4.paradoxwikis.com/Modifier_list

File: localisation/Idea_Variation_ideas_l_german.yml [hunk 1]
```diff
@@ -143,22 +143,24 @@
...
  republik3: "Grundrechte"
  republik3_desc: "Indem wir den Bürgern gewisse Grundrechte zubilligen, verringern wir die Gefahr von Bauernaufständen."
- republik5: "Republikanische Tradition"
- republik5_desc: "Unsere Republik ist über die Jahre gewachsen und in sich gereift, sodass sie nicht viel erschüttern kann. \nErmöglicht §GHofiert die Bürger§W Stand Interaktion"
  republik4: "Freie Untertanen"
  republik4_desc: "So befriedigend es auch sein mag, einem Diener eine ordentliche Tracht Prügel zu verpassen – möchte man ihren Arbeitseinsatz erhöhen, so ist dies nicht die beste Methode. Ein Naturphilosoph hatte einmal die Idee, dass die Hauptmotivation der Menschen das Streben nach Profit sei. Wenn wir Geldzahlungen durch Bezahlung in Naturalien ersetzen, hätten die Bauern einen Anreiz ihre Produktion zu steigern."
- republik1: "Diplomatische Mission"
- republik1_desc: "Wir müssen uns als Republik stärker diplomatisch absichern als manche Monarchien. Denn unsere Art zu Leben stellt Chance und Gefahr zu gleich da."
+ republik5: "Die Macht eines Diplomaten"
+ republik5_desc: "Ein einziger Diplomat am richtigen Ort und zur richtigen Zeit kann den Lauf der Geschichte verändern"
```
Comment (stackpoint) [inline localisation/Idea_Variation_ideas_l_german.yml:158]: Are you German or did you copy/paste this from somewhere else?

File: common/estates/03_burghers.txt [hunk 2]
```diff
@@ -1798,7 +1798,7 @@ estate_burghers =
 		}
 		
 		trigger = {
-			has_idea = republik5
+			has_idea = republik7
```
Comment (stackpoint) [inline common/estates/03_burghers.txt:1801]: Please be aware that the Jains has this government interaction too.

**Review by stackpoint — CHANGES_REQUESTED** [review]

**Comment by stackpoint** [comment]
Adding a diplomat in there seems fine but rearranging the tree has it's own problems as you can see. I also didn't get pinged when you updated a commit for some reason, might have to make a comment or request another review.

File: localisation/Idea_Variation_ideas_l_german.yml [hunk 3]
```diff
@@ -143,22 +143,24 @@
  monarchie7_desc: "Falls es zum Krieg kommt stellt unser Adel örtliche Millizen auf, die bei der Verteidigung des Landes helfen. \nErmöglicht §GVerpflichtet die lokale Milliz§W Stand Interaktion"
  monarchie5: "Erhöhung der Kronautorität"
  monarchie5_desc: "Durch Erhöhung unserer Autorität ist es uns einfacher die Kontrolle im Land zu behalten und dessen Stabilität zu sichern."
+ 
  republik0: "Republikanische Ideen"
  republik0_desc: "Die republikanische Regierungsform wird von unserem Staat verkörpert."
- republik7: "Römische Vorväter"
- republik7_desc: "Wir können viel von den Römern lernen, die vor uns kamen. Sie schafften es in Zeiten der Republik ein großes Reich aufzubauen und es relativ zentral zu verwalten."
+ republik1: "Diplomatische Mission"
+ republik1_desc: "Wir müssen uns als Republik stärker diplomatisch absichern als manche Monarchien. Denn unsere Art zu Leben stellt Chance und Gefahr zu gleich da."
```
Comment (Grociu) [inline localisation/Idea_Variation_ideas_l_german.yml:150]: We are generally no longer updating German localization, only the default English. If this is a lot of extra effort for you, you can safely skip this next time.

File: localisation/Idea_Variation_ideas_l_english.yml [hunk 4]
```diff
@@ -145,22 +145,24 @@ l_english:
...
  republik2_desc: "We know from reading the accounts of Roman times that a citizen army of free men is far more willing to fight than those who are forced. We must make our peasantry fairly free and they will fight to defend their freedom."
  republik3: "Bill of Rights"
  republik3_desc: "By granting the citizens certain fundamental rights, we reduce the likelihood of a peasant uprising."
- republik5: "Republican Tradition"
- republik5_desc: "As a long standing republic, our government will more easily face and endure challenges. \nEnable §GPaying court to the Burghers§W Estate Interaction"
  republik4: "Free Subjects"
  republik4_desc: "Although giving a servant a damn good thrashing might be satisfying, it is not the best way to actually get them to work harder. Some natural philosopher has hit on the idea of making profit the prime motivation for people. If we replace monetary rents with payment in kind, our peasants will be incentivized to produce more."
- republik1: "Diplomatic Mission"
- republik1_desc: "We need to play the diplomatic game to ensure our freedom. Because our way of life is both a chance and a threat to our neighbors."
+ republik5: "The Power of a Diplomat"
```
Comment (Grociu) [inline localisation/Idea_Variation_ideas_l_english.yml:159]: (Suggestion) As a running gag, whenever a function of an idea is changed, I've been using [alliteration](https://en.wikipedia.org/wiki/Alliteration) in individual Idea names when possible. In that vein, here's a list of suggested idea names for this:

- Dynamic Diplomat
- Effective Envoy
- Powerful Plenipotentiary (my current favorite)
- Convincing Conciliator
- Persuasive Peacemaker

... and so on, feel free to come up with one yourself, it's fun.

Comment (stackpoint) [reply localisation/Idea_Variation_ideas_l_english.yml:159]: As long as it's written in English I'm happy.

Comment (ComradDoge) [reply localisation/Idea_Variation_ideas_l_english.yml:159]: Is plenipotentiary even English at this point ? Although I like it for jokes

Comment (Grociu) [reply localisation/Idea_Variation_ideas_l_english.yml:159]: https://www.vocabulary.com/dictionary/plenipotentiary 
Ha, it works well here doesn't it :)

**Comment by Grociu** [comment]
Another suggestion for future workflow. I noticed you're pushing from master. Someone taught me when I did my first GitHub collaboration this order for Pull Requests:
- on your local machine copy of the repository, make a new git branch
- commit all changes to that branch locally
- push that branch with the changes to your GitHub fork (usually origin)
- submit a Pull Request from your GitHub fork of the repository. 

You'll be merging your own branch into the common master instead of master to master directly. This will assure, that even if the common master branch (the upstream) changes, your branch is truly independent of those changes, and you can still pull the upstream master into your own without trouble. In addition any requested changes after the initial commit you just commit to your branch and they show up in GitHub pull request normally. In addition you can work on multiple issues at the same time and have a branch for each.

I am in no means an expert on the matter and still learning git and GitHub myself, but this is the observed good practice :)

**Comment by ComradDoge** [comment]
Thank you very much. I literally started learning GitHub yesterday so the terms and how everything goes still confuses me but will get used to it with time.

**Comment by Grociu** [comment]
Yeah I forget that all these terms are jargon. Here's an article that explains it in plain English, there are probable better resources out there depending on your level of knowledge, but this is an amazing starter:
https://blog.red-badger.com/2016/11/29/gitgithub-in-plain-english
I got to this article by using this course on Odin Project: 
https://www.theodinproject.com/courses/web-development-101/lessons/introduction-to-git

**Comment by stackpoint** [comment]
Okay, I will merge this in and modify the Jains
